projects/1.21/assets/gardens-of-the-dead/gardens_of_the_dead/lang/zh_cn.json

```
@@ -0,0 +1,52 @@
+{
+    "gardens_of_the_dead.creative_tab": "死灵花园",
+    "biome.gardens_of_the_dead.soulblight_forest": "枯魂森林",
+    "biome.gardens_of_the_dead.whistling_woods": "鸣怨竹林",
+    "block.gardens_of_the_dead.soul_spore": "灵魂菌索",
+    "block.gardens_of_the_dead.glowing_soul_spore": "发光灵魂菌索",
+    "block.gardens_of_the_dead.soulblight_fungus": "枯魂菌",
+    "block.gardens_of_the_dead.soulblight_sprouts": "枯魂菌簇",
+    "block.gardens_of_the_dead.blistercrown": "疱冠菌",
+    "block.gardens_of_the_dead.tall_blistercrown": "高疱冠菌簇",
+    "block.gardens_of_the_dead.whistlecane": "鸣怨竹",
+    "block.gardens_of_the_dead.soulblight_stem": "枯魂菌柄",
+    "block.gardens_of_the_dead.stripped_soulblight_stem": "去皮枯魂菌柄",
+    "block.gardens_of_the_dead.soulblight_hyphae": "枯魂菌核",
+    "block.gardens_of_the_dead.stripped_soulblight_hyphae": "去皮枯魂菌核",
+    "block.gardens_of_the_dead.blightwart_block": "枯魂疣块",
+    "block.gardens_of_the_dead.potted_soul_spore": "枯魂菌索盆栽",
+    "block.gardens_of_the_dead.potted_glowing_soul_spore": "发光枯魂菌索盆栽",
+    "block.gardens_of_the_dead.potted_soulblight_fungus": "枯魂菌盆栽",
+    "block.gardens_of_the_dead.potted_soulblight_sprouts": "枯魂菌簇盆栽",

```

```suggestion
    "block.gardens_of_the_dead.potted_soulblight_sprouts": "枯魂苗盆栽",
```
可以和这个对照一下
https://zh.minecraft.wiki/w/%E4%B8%8B%E7%95%8C%E8%8B%97

---

> 可以和这个对照一下 https://zh.minecraft.wiki/w/%E4%B8%8B%E7%95%8C%E8%8B%97

嘶，这玩意和下界苗完全不一样，整体样子类似菌素那一类，但空手打了之后不会像它们一样直接变成掉落物。

---

下界苗也是空手破坏不会掉落，使用剪刀破坏才掉落的，名称也符合，大概还是按照原版来吧

---

> 下界苗也是空手破坏不会掉落，使用剪刀破坏才掉落的，名称也符合，大概还是按照原版来吧

OK